<launch>
    <arg name="map" default="$(find mushr_rhc_ros)/maps/imdang3F.yaml" />
    <arg name="c2g_map" default="$(find mushr_rhc_ros)/maps/imdang3F.yaml" />
    <arg name="c2g_map1" default="$(find mushr_rhc_ros)/maps/imdang3F.yaml" />
    <arg name="map1" default="$(find mushr_rhc_ros)/maps/imdang3F.yaml" />

    <node pkg="map_server" name="map_server" type="map_server" args="$(arg map)" />
    <node pkg="map_server" name="c2g_map" type="map_server" args="$(arg c2g_map)">
        <remap from="/static_map" to="/c2g/static_map" />
        <remap from="/map_metadata" to="/c2g/map_metadata" />
        <remap from="/map" to="/c2g/map" />
    </node>

    <param name="map_file" value="$(arg c2g_map)" />

    <include file="$(find mushr_pf)/launch/real.launch" />

</launch>
